segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -205,8 +215,55 @@ private ByteBuffer copyData(List<ReadResultEntryContents> contents) {
         }
         return data;
     }
+    
+    @Override
+    public void updateSegmentAttribute(UpdateSegmentAttribute updateSegmentAttribute) {
+        long requestId = updateSegmentAttribute.getRequestId();
+        String segmentName = updateSegmentAttribute.getSegmentName();
+        UUID attributeId = updateSegmentAttribute.getAttributeId();
+        long newValue = updateSegmentAttribute.getNewValue();
+        long expectedValue = updateSegmentAttribute.getExpectedValue();
+        val update = new AttributeUpdate(attributeId, AttributeUpdateType.ReplaceIfEquals, newValue, expectedValue);
+        segmentStore.updateAttributes(segmentName, Collections.singletonList(update), TIMEOUT).whenComplete((v, e) -> {
+            if (e == null) {
+                connection.send(new SegmentAttributeUpdated(requestId, true));
+            } else {
+                if (ExceptionHelpers.getRealException(e) instanceof BadAttributeUpdateException) {
```

Comment:
I just want to check that this exception is actually logged somewhere. I know that for a while we were not really logging server-side exceptions caused by bad input.